tasks/18_sshdconfig.yml

```diff
@@ -41,12 +41,12 @@
   notify:
     - restart sshd
 
-# Only the RedHat instances fail the related compliance check
-- name: Set ssh ciphers, MAC algorithms, and kex algorithms (RedHat)
+- name: Set ssh ciphers, MAC algorithms, and kex algorithms
   become: yes
   lineinfile:
     line: "{{ item.name }} {{ item.value }}"
-    path: /etc/crypto-policies/back-ends/opensshserver.config
+    # RedHat (or at least Fedora) stores this data in an odd location
+    path: "{{ (ansible_os_family == 'RedHat') | ternary('/etc/crypto-policies/back-ends/opensshserver.config', '/etc/ssh/sshd_config') }}"
```

Comment:
I love a good ternary!

Reply:
That's a great idea, but it doesn't make sense to use `wheel` here; to satisfy the compliance warning we must use an _empty_ group for this purpose.

Reply:
> I had one thought to add which is would it make sense to use the group name `wheel` (instead of `sugroup`) so this role would follow how we grant `sudoer` access on systems? That would mirror [cisagov/ansible-role-dev-ssh-access](https://github.com/cisagov/ansible-role-dev-ssh-access/blob/1e443b285329650e0ae9e0798a5ae79df2939ab6/tasks/main.yml#L11-L17) creating a `wheel` group for password-less `sudo`.

That's a great idea, but it doesn't make sense to use `wheel` here; to satisfy the compliance warning we must use an _empty_ group for this purpose.

Reply:
Ah, right. I always forget that `su` is essentially deprecated vs `sudo`. Empty group to discourage `su` use makes sense 👍